examples/flutter_gallery/lib/demo/cupertino/cupertino_alert_demo.dart

```diff
@@ -15,181 +14,162 @@ class CupertinoAlertDemo extends StatefulWidget {
+                            Navigator.pop(context, 'Disallow');
+                          },
+                        ),
+                        CupertinoDialogAction(
+                          child: const Text('Allow'),
+                          onPressed: () {
+                            Navigator.pop(context, 'Allow');
+                          },
+                        ),
+                      ],
                     ),
-                    CupertinoActionSheetAction(
-                      child: const Text('Cannolis'),
-                      onPressed: () {
-                        Navigator.pop(context, 'Cannolis');
-                      },
+                  );
+                },
+              ),
+              const Padding(padding: EdgeInsets.all(8.0)),
+              CupertinoButton.filled(
+                child: const Text('Alert with Buttons'),
+                padding: const EdgeInsets.symmetric(vertical: 16.0, horizontal: 36.0),
+                onPressed: () {
+                  showDemoDialog<String>(
+                    context: context,
+                    child: const CupertinoDessertDialog(
+                      title: Text('Select Favorite Dessert'),
+                      content: Text('Please select your favorite type of dessert from the '
+                        'list below. Your selection will be used to customize the suggested '
+                        'list of eateries in your area.'),
                     ),
-                    CupertinoActionSheetAction(
-                      child: const Text('Trifle'),
-                      onPressed: () {
-                        Navigator.pop(context, 'Trifle');
-                      },
+                  );
+                },
+              ),
+              const Padding(padding: EdgeInsets.all(8.0)),
+              CupertinoButton.filled(
+                child: const Text('Alert Buttons Only'),
+                padding: const EdgeInsets.symmetric(vertical: 16.0, horizontal: 36.0),
+                onPressed: () {
+                  showDemoDialog<String>(
+                    context: context,
+                    child: const CupertinoDessertDialog(),
+                  );
+                },
+              ),
+              const Padding(padding: EdgeInsets.all(8.0)),
+              CupertinoButton.filled(
+                child: const Text('Action Sheet'),
+                padding: const EdgeInsets.symmetric(vertical: 16.0, horizontal: 36.0),
+                onPressed: () {
+                  showDemoActionSheet<String>(
+                    context: context,
+                    child: CupertinoActionSheet(
+                      title: const Text('Favorite Dessert'),
+                      message: const Text('Please select the best dessert from the options below.'),
+                      actions: <Widget>[
+                        CupertinoActionSheetAction(
+                          child: const Text('Profiteroles'),
```

Comment:
Important note, thanks to Google. A profiteroles is a cream puff. The more you know.